node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/sync/SyncFileInfo.java

```diff
@@ -82,12 +82,14 @@ public static Optional<SyncFileInfo> forLogFile(URI uri, Path logFile, boolean r
             compression = filename.endsWith(".zst") ? Compression.NONE : Compression.ZSTD;
             if (rotatedOnly && compression != Compression.NONE)
                 dir = null;
+            else if (filename.matches(".*\\.metrics-proxy\\..*")) // See AccessLogComponent.java for filename.
```

Comment:
Maybe `filename.contains(".metrics-proxy.")` to avoid regex?

Reply:
Of course ...

Reply:
```suggestion
            else if (filename.contains(".metrics-proxy.")) // See AccessLogComponent.java for filename.
```